MaterialLib/MPL/Properties/CapillaryPressureSaturation/SaturationVanGenuchten.h

```diff
@@ -16,48 +16,37 @@ class Medium;
 class Phase;
 class Component;
 /**
- *   \brief The van Genuchten capillary pressure model:
+ * \brief The van Genuchten capillary pressure model.
  *
- *   \f[p_c(S)=p_b (S_e^{-1/m}-1)^{1-m}\f]
- *   with
- *   \f[S_e=\frac{S-S_r}{S_{\mbox{max}}-S_r}\f]
- *   where
- *    \f{eqnarray*}{
- *       &p_b&            \mbox{ entry pressure,}\\
- *       &S_r&            \mbox{ residual saturation,}\\
- *       &S_{\mbox{max}}& \mbox{ maximum saturation,}\\
- *       &m \in [0,1) &        \mbox{ exponent.}\\
- *    \f}
+ * The van Genuchten capillary pressure model (\cite Genuchten1980) is:
+ * \f[p_c(S)=\frac{1}{\alpha} (S_\text{eff}^{-1/m}-1)^{1-m}\f]
+ * with effective saturation defined as
+ * \f[S_\text{eff}=\frac{S-S_r}{S_{\text{max}}-S_r}.\f]
+ * Above, \f$S_r\f$ and \f$S_{\text{max}}\f$ are the residual and the maximum
+ * saturations.
+ * The exponent \f$m \in (0,1)\f$ and the parameter \f$\alpha\f$ are given by
+ * the user.
```

Comment:
Practically, the entry pressure \f$p_d\f$ is introduced to replace \f$ \frac{\rho_i g}{\alpha}\f$.  In the current implementation, the entry pressure \f$p_d\f$ is used.